resources/views/spa.blade.php

```diff
@@ -24,13 +26,31 @@
 	<meta name="theme-color" content="#ffffff">
 	<meta name="msapplication-config" content="favicon/browserconfig.xml" />
 
+	<script>
+    var m_id = '<?php echo $measurement_id; ?>';
+    var src_url = 'https://www.googletagmanager.com/gtag/js?id=' + m_id; // Appending m_id to URL
+	//alert(src_url);
+	</script>
+
+	<!-- Global site tag (gtag.js) - Google Analytics -->
+	<?php 
+	if($measurement_id != 'None') {
+	  echo "<script async src=src_url></script>
+	        <script>
+	          window.dataLayer = window.dataLayer || [];
+	          function gtag(){dataLayer.push(arguments);}
+	          gtag('js', new Date());
+	          gtag('config', m_id);
+	        </script>";
+	}
+	?>
```

Comment:
just to let you know, instead than declare the PHP block like you did and "echo" html, in the PHP blade you can use a much cleaner syntax like explained in this link: https://stackoverflow.com/questions/40122633/if-else-condition-in-blade-file-laravel-5-3

@if($measurement_id !== 'None')         
      <script> your script tag</script>    
@endif